src/istep/edbgIstep.C

```diff
@@ -232,14 +232,10 @@ const edbgIPLTable::edbgIStep_t edbgIPLTable::cv_edbgIStepTable[] =
   {18,  13,                            "attn_listenall",                   EDBG_ISTEP_NOOP},
   {19,   1,                                 "prep_host",                   EDBG_ISTEP_NOOP},
   {20,   1,                         "host_load_payload",                   EDBG_ISTEP_HOST},
-  {20,   2,                        "host_load_complete",                   EDBG_ISTEP_HOST},
-  {21,   1,                         "host_micro_update",                   EDBG_ISTEP_NOOP},
-  {21,   2,                        "host_runtime_setup",                   EDBG_ISTEP_HOST},
+  {21,   1,                        "host_runtime_setup",                   EDBG_ISTEP_HOST},
+  {21,   2,                         "host_micro_update",                   EDBG_ISTEP_NOOP},
```

Comment:
use "dummy_istep_name" instead of host_micro_update

Reply:
Done